content/docs/command-reference/exp/run.md

```diff
@@ -108,8 +109,11 @@ The `--queue` option lets you create an experiment as usual, except that nothing
 is actually run. Instead, the experiment is put in a wait-list for later
 execution. `dvc exp show` will mark queued experiments with an asterisk `*`.
 
-Use `dvc exp run --run-all` to process this queue. Note that if the queued
-experiments use checkpoints, `--run-all` implies `--reset` (restarts them).
+Note that for experiments which use checkpoints, queuing an experiment implies
+`--reset` unless an existing checkpoint to resume is explicitly provided via
+`--rev`.
+
+Use `dvc exp run --run-all` to process this queue.
 
 Adding `-j` (`--jobs`), experiment queues can be run in parallel for better
 performance. This creates a temporary workspace copy for each subprocess (in
```

Comment:
```suggestion
> Note that queuing an experiment that uses checkpoints implies `--reset`
> unless `--rev` is used.

Use `dvc exp run --run-all` to process the queue. Adding `-j` (`--jobs`),
experiment queues can be run in parallel for better performance. This creates a
temporary workspace copy for each subprocess (in
```

The 2nd p needs wrapping with the remaining line but I couldn't include it in the suggestion.